core/templates/dev/head/pages/exploration_editor/editor_tab/UnresolvedAnswersOverviewDirective.js

```diff
@@ -150,6 +152,14 @@ oppia.directive('unresolvedAnswersOverview', [
                   // Inject RulesService dynamically.
                   var rulesService = $injector.get(rulesServiceName);
 
+                  var RESPONSE_CONFIRMED_TOAST_MESSAGE = (
+                    'Response of the answer has been confirmed');
```

Comment:
`The answer "{{short snippet of answer}}..." has been successfully trained.`

(If the answer is sufficiently short, don't need the "...". But including a snippet helps distinguish the messages for different answers, especially if the creator is working quite quickly.

Reply:
The answer snippet would work for interactions like text, numeric input only. It wouldn't work for code repl and similar complex input interaction.